pkg/daemon/update.go

```diff
@@ -467,6 +467,7 @@ func (dn *Daemon) applyOSChanges(oldConfig, newConfig *mcfgv1.MachineConfig) (re
 func calculatePostConfigChangeActionFromFileDiffs(oldIgnConfig, newIgnConfig ign3types.Config) (actions []string) {
 	filesPostConfigChangeActionNone := []string{
 		"/var/lib/kubelet/config.json",
+		"/etc/kubernetes/kubelet-ca.crt",
```

Comment:
This by itself probably won't do anything. I've never observed this to change without `/etc/kubernetes/ca.crt` also changing.

Reply:
`/etc/kubernetes/ca.crt` gets rendered out from the bootstrap to the masters... I do not see any processes reading ca.crt on a fully installed master. The kubelet does not use ca.crt.

I added /etc/kubernetes/ca.crt into the list.

Reply:
With David's patch we can just keep kubelet-ca.crt...